<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendItem</class>
 <widget class="QWidget" name="FriendItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_tx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>uid:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
